app/models/event.rb

```diff
@@ -16,9 +19,13 @@ class Event < ActiveRecord::Base
   validates :slug, uniqueness: true
   validate :invitability, if: :invitable?
   validates :coach_spaces, :student_spaces, numericality: true
+  validate :sponsors_uniqueness
+  validate :bronze_sponsors_uniqueness
+  validate :silver_sponsors_uniqueness
+  validate :gold_sponsors_uniqueness
```

Comment:
It seems to me like a sponsor is probably unique across all levels. Maybe @KimberleyCook can clarify is someone can be a gold and a bronze sponsor, for example.

I think that could reduce the amount of code needed as well

Reply:
Update: I just saw `sponsors_uniqueness` above which does what I was expecting. In that case I don't think validating inside levels hurts

Reply:
the validation doesn't discriminate between levels, so it will fail either if the duplication is at the same level or in different levels